### PR TITLE
Install MCG addon on CI cluster

### PR DESCRIPTION
**Pull request by rexagod**
Install MCG addon on CI cluster for integration testing.

### CODE REVIEW

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -2,6 +2,90 @@
...
+    export OS=$(uname | awk '{print tolower($0)}')
+    export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.20.0
+    curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+    chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+}
+
+function installOperator {
+    export TAG=ci
+    export IMG=quay.io/rexagod/mcg-osd-deployer:$TAG
+    export BUNDLE_IMG=quay.io/rexagod/mcg-osd-deployer:bundle-$TAG
```
Comment (naveenpaul1) [inline test-prow-e2e.sh:16]: I think here we need to provide mcg-osd repo url,

Comment (rexagod) [reply test-prow-e2e.sh:16]: That was my first thought as well, but I saw that the `bundle`, `index` and `prometheus` images were last updated on 27th April. Should I still use https://quay.io/repository/osd-addons/mcg-osd?

Comment (cloudbehl) [reply test-prow-e2e.sh:16]: I agree we should use mcg-osd. Let's update the Images if required but we should use that only.

Comment (rexagod) [reply test-prow-e2e.sh:16]: @naveenpaul1 Do you have push access to this? I haven't heard back from the MT-SRE team on my request to grant me those rights, but as long as someone from our team can push images for testing, I can go ahead and make this change, PLMK, thanks.

Comment (naveenpaul1) [reply test-prow-e2e.sh:16]: I have push access to `osd-addons/mcg-osd` I can push the image

**Comment by rexagod** [comment]
cc @SanjalKatiyar

File: test-prow-e2e.sh (hunk 1 above)
Comment (naveenpaul1) [reply test-prow-e2e.sh]: /LGTM

**Comment by rexagod** [comment]
Do not merge, will send a newer patch that will remove OperatorSDK from the prerequisites.

**Comment by rexagod** [comment]
Verified on a fresh cluster.

**Pull request merged**